src/gtk/widgets/recent_info.rs

```diff
@@ -141,23 +142,22 @@ impl RecentInfo {
 
     pub fn has_group(&self, group_name: &str) -> bool {
         unsafe {
-            to_bool(
-                ffi::gtk_recent_info_has_group(GTK_RECENT_INFO(self.unwrap_widget()),
-                                               group_name.borrow_to_glib().0))
+            to_bool(ffi::gtk_recent_info_has_group(GTK_RECENT_INFO(self.unwrap_widget()),
+                group_name.borrow_to_glib().0))
         }
```

Comment:
It makes sense to integrate a code clean up within this file, because this pull request already modifies this particular file. However, such changes can always be reverted. The `group_name` argument is passed to `ffi::gtk_recentinfo_has_group`, which is clearly shown by any editor that shows matching parentheses. Besides, the problem with alignment of arguments is that there are plenty of cases where you would have to trade a lot of horizontal space for vertical space, for no particular reason other than decorative purposes.